### PR TITLE
housekeeping: Added Central Package Versioning

### PR DESCRIPTION
**Pull request by RLittlesII**

### CODE REVIEW

File: directory.build.props [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<Project>
+    <PropertyGroup>
+        <Authors>ReactiveUI Association Inc</Authors>
+        <Company>ReactiveUI Association Inc</Company>
```
Comment (glennawatson) [inline directory.build.props:4]: Lets keep the /src structure for source code

File: directory.build.props [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<Project>
+    <PropertyGroup>
+        <Authors>ReactiveUI Association Inc</Authors>
```
Comment (glennawatson) [inline directory.build.props:3]: Lets put this under /src

File: directory.build.targets [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Project>
+  <ItemGroup>
+    <PackageReference
+            Include="Microsoft.NETFramework.ReferenceAssemblies"
```
Comment (glennawatson) [inline directory.build.targets:5]: Don't think we need this at this time.

File: directory.build.targets [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+            IncludeAssets="analyzers;build;buildMultitargeting;buildTransitive"
+            PrivateAssets="All"
+        />
+    <PackageReference Include="Microsoft.SourceLink.GitHub" IncludeAssets="analyzers;build;buildMultitargeting;buildTransitive" PrivateAssets="All" />
```
Comment (glennawatson) [inline directory.build.targets:9]: SourceLink can be under Directory.build.props file, doesn't have to be under .targets file

File: directory.packages.props [hunk 5]
```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Project>
+  <ItemGroup>
+    <PackageVersion Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.2" />
+    <PackageVersion Include="Microsoft.SourceLink.GitHub" Version="1.1.1" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageVersion Include="Nuke.Common" Version="6.0.1" />
+    <PackageVersion Include="NuGet.CommandLine" Version="5.8.1" />
```
Comment (glennawatson) [inline directory.packages.props:9]: Let's remove unused dependencies not needed yet. Eg the Nuke/Resharper stuff

**Review by glennawatson — CHANGES_REQUESTED** [review]

**Comment by glennawatson** [comment]
btw those compile errors be useful to fix in tests at some point also.

File: directory.build.props (hunk 2 above)
Comment (RLittlesII) [reply directory.build.props]: moved

File: directory.build.targets (hunk 3 above)
Comment (RLittlesII) [reply directory.build.targets]: removed

File: directory.build.targets (hunk 4 above)
Comment (RLittlesII) [reply directory.build.targets]: copy paste error.

File: directory.packages.props (hunk 5 above)
Comment (RLittlesII) [reply directory.packages.props]: removed

**Review by glennawatson — APPROVED** [review]

**Pull request merged**